src/javascript/lib/landing_pages/common.js

```diff
@@ -48,15 +50,15 @@ Math.easeInOutQuad = function (current_time, start_value, change_in_value, durat
 };
 
 function getParamValue(url, key) {
-    var regex   = new RegExp('[?&]' + key + '(=([^&#]*)|&|#|$)');
-    var results = regex.exec(url);
+    let regex   = new RegExp('[?&]' + key + '(=([^&#]*)|&|#|$)');
```

Comment:
@kellybinary Just to follow eslint setting in other binary-static pages ... could you please use `const` for constant variables instead of `let`.

Reply:
sure @aminroosta 👍